### ISSUE TITLE
Scam 90% performance steal

### ISSUE THREAD

**Issue by Flex-Mini** [UnamSanctam/SilentCryptoMiner#752]
after couple days it mines everything 10 times slower, probably 90% mined shit comes to his wallets

dont use this shit, just use normal xmrig its easy to set it up run silently, and for antivirus detectionions all miners get detected

**Comment by Nobellium1** [comment]
This is open source project where you can clearly see if he's stealing anything or not...

**Comment by JosepP2** [comment]
> This is open source project where you can clearly see if he's stealing anything or not...

idiot child, You don't know shit. 
Connect your bots to your localhost 127.0.0.1 best idea.

**Comment by JosepP2** [comment]
> after couple days it mines everything 10 times slower, probably 90% mined shit comes to his wallets
> 
> dont use this shit, just use normal xmrig its easy to set it up run silently, and for antivirus detectionions all miners get detected

Don't talk, if you dont have proof please, there lot of people using this project. i know there is paid applications cutting your %50 even you pay them. also you depentent with their server's imagine if their server's gone you cannot login your panel.

**Comment by UnamSanctam** [comment]
> after couple days it mines everything 10 times slower, probably 90% mined shit comes to his wallets
> 
> dont use this shit, just use normal xmrig its easy to set it up run silently, and for antivirus detectionions all miners get detected

You can clearly just test it and see that that's not the case, I don't really understand what you stand to gain by lying about such a thing? Show a screenshot or anything to show what you're talking about. If you use the computer name as the worker name for the pool you'll see the statistics for each computer and you'll see that you're always getting the same average hashrate (same as a normal xmrig) no matter how long you mine, you can even set up the web panel and compare it to the pool/normal statistics to disprove what you're saying as well.

**Comment by Flex-Mini** [comment]
> > after couple days it mines everything 10 times slower, probably 90% mined shit comes to his wallets
> > dont use this shit, just use normal xmrig its easy to set it up run silently, and for antivirus detectionions all miners get detected
> 
> You can clearly just test it and see that that's not the case, I don't really understand what you stand to gain by lying about such a thing? Show a screenshot or anything to show what you're talking about. If you use the computer name as the worker name for the pool you'll see the statistics for each computer and you'll see that you're always getting the same average hashrate (same as a normal xmrig) no matter how long you mine, you can even set up the web panel and compare it to the pool/normal statistics to disprove what you're saying as well.

idk wanted to see if you would delete the issue to confirm my suspicions

i have both web panel setup and i look on pool site dashboard, same pc dropped performance 10x.

what is the problem then? hashrates are too low

**Comment by UnamSanctam** [comment]
What CPU and what hashrate are you getting? Is it just that one computer? One thing I could maybe think of is that it's out of RAM at the moment so there's no huge pages. You could try changing it to some other algorithm temporarily like GhostRider (doesn't really use any RAM) though the issue is that you don't know the standard hashrate for GhostRider on that CPU.

**Comment by JosepP2** [comment]
> > > after couple days it mines everything 10 times slower, probably 90% mined shit comes to his wallets
> > > dont use this shit, just use normal xmrig its easy to set it up run silently, and for antivirus detectionions all miners get detected
> > 
> > 
> > You can clearly just test it and see that that's not the case, I don't really understand what you stand to gain by lying about such a thing? Show a screenshot or anything to show what you're talking about. If you use the computer name as the worker name for the pool you'll see the statistics for each computer and you'll see that you're always getting the same average hashrate (same as a normal xmrig) no matter how long you mine, you can even set up the web panel and compare it to the pool/normal statistics to disprove what you're saying as well.
> 
> idk wanted to see if you would delete the issue to confirm my suspicions
> 
> i have both web panel setup and i look on pool site dashboard, same pc dropped performance 10x.
> 
> what is the problem then? hashrates are too low

I assume Hashrates of your rig dropped 10x... not miner's calculation.
Thats sad when you talk like this when you dont have proof, because we all using his miner. and im using since last year..
my hashrate didn't drop 10x.. keep in mind, if you stop spreading, you will lose your bots day by day. 
because they resetting pc or windows.

**Comment by Flex-Mini** [comment]
> > > > after couple days it mines everything 10 times slower, probably 90% mined shit comes to his wallets
> > > > dont use this shit, just use normal xmrig its easy to set it up run silently, and for antivirus detectionions all miners get detected
> > > 
> > > 
> > > You can clearly just test it and see that that's not the case, I don't really understand what you stand to gain by lying about such a thing? Show a screenshot or anything to show what you're talking about. If you use the computer name as the worker name for the pool you'll see the statistics for each computer and you'll see that you're always getting the same average hashrate (same as a normal xmrig) no matter how long you mine, you can even set up the web panel and compare it to the pool/normal statistics to disprove what you're saying as well.
> > 
> > 
> > idk wanted to see if you would delete the issue to confirm my suspicions
> > i have both web panel setup and i look on pool site dashboard, same pc dropped performance 10x.
> > what is the problem then? hashrates are too low
> 
> I assume Hashrates of your rig dropped 10x... not miner's calculation. Thats sad when you talk like this when you dont have proof, because we all using his miner. and im using since last year.. my hashrate didn't drop 10x.. keep in mind, if you stop spreading, you will lose your bots day by day. because they resetting pc or windows.

ik bro, how much bots u have?

**Comment by JosepP2** [comment]
i have 3ghs+ ethereum, 700mhsetc, 700kh xmr. i never lost any bots from him, 
before i had issues because the project was new, and he giving best effort to make this project alive.
all of you donate a little bit before talk, the child of top Nob* creating 5x issue daily, i bet he didnt donate a bucks.

**Comment by Flex-Mini** [comment]
> What CPU and what hashrate are you getting? Is it just that one computer? One thing I could maybe think of is that it's out of RAM at the moment so there's no huge pages. You could try changing it to some other algorithm temporarily like GhostRider (doesn't really use any RAM) though the issue is that you don't know the standard hashrate for GhostRider on that CPU.

rechecked so now i only got one active bot with AMD A8-3850 APU mining on 90H/s, on max it should get 400H/s, my max cpu not idle is set 50% so i should be getting at least 200H/s

other offline bots

**Comment by UnamSanctam** [comment]
Sounds like it might be using 1 core of its 4 cores, it's also possible that there's not enough RAM or something similar (I have seen computers with that series of CPU with 4GB of RAM), it's also possible that only one core was "available" so it only uses that since xmrig is set to yield (to stop any lag). RandomX is relatively quite a complicated and demanding algorithm so there's many different things which affect the hashrate, you can try setting the "Max CPU" at a higher percentage for that miner (create a new configuration in the web panel with a higher CPU and apply it to that miner) and see what the difference is. You could also try another algorithm to see if it's due to the relatively high RandomX requirements but the problem is that I can only find benchmarks for RandomX online for that CPU.

**Comment by Nobellium1** [comment]
> > This is open source project where you can clearly see if he's stealing anything or not...
> 
> idiot child, You don't know shit. Connect your bots to your localhost 127.0.0.1 best idea.

Holy shit you know what a localhost is?! Good job monkey, here's a banana! Now get the fuck out of here

**Comment by JosepP2** [comment]
> > > This is open source project where you can clearly see if he's stealing anything or not...
> > 
> > 
> > idiot child, You don't know shit. Connect your bots to your localhost 127.0.0.1 best idea.
> 
> Holy shit you know what a localhost is?! Good job monkey, here's a banana! Now get the fuck out of here

Yes you have directed your bots to 127.0.0.1 and asking unam; where is my bots? russian bastard child.
didnt even know pool's share interval, and talking like boss here. fuck off, suck your mom' and grow up.

**Comment by Nobellium1** [comment]
First of all I obviously wouldnt be stupid as you to connect them to localhost and why the fuck do you care what am I asking unam? None of your bussiness you fuckin stalker. Also I'm not russian and with your english I'm assuming you might be either russsian or african.
Its funny how you made those stories inside your dumbfuckin head and then pretty much just talk random stuff here. 
Get the fuck out of your mother's basement fucktarded braindead child

**Comment by Nobellium1** [comment]
You're one of those scammers who wants to give their wallet for people to mine after saying that you have some superior miner that is self coded by you and when someone has over 2 braincells not to fall for your shitty scam, you start crying here in the fuckin github repository... Get a fuckin life mongoloid

**Comment by Flex-Mini** [comment]
> You're one of those scammers who wants to give their wallet for people to mine after saying that you have some superior miner that is self coded by you and when someone has over 2 braincells not to fall for your shitty scam, you start crying here in the fuckin github repository... Get a fuckin life mongoloid

stfu 14year old nerd

**Comment by Nobellium1** [comment]
> > You're one of those scammers who wants to give their wallet for people to mine after saying that you have some superior miner that is self coded by you and when someone has over 2 braincells not to fall for your shitty scam, you start crying here in the fuckin github repository... Get a fuckin life mongoloid
> 
> stfu

Thats the fact. You can gtfo too

**Comment by harry133712** [comment]
how can i setup ghostrider in my pastebin ? and also is it possible to mine the ghostrider(Raptoreum) and get paid out in xmr on xmr oocean?

**Comment by UnamSanctam** [comment]
> how can i setup ghostrider in my pastebin ? and also is it possible to mine the ghostrider(Raptoreum) and get paid out in xmr on xmr oocean?

For MoneroOcean GhostRider you can find an example one here: https://rentry.co/vnspf/raw which works.

**Comment by harry133712** [comment]
so i can leave my xmr wallet and only have to change port and password to mine ghostrider?

**Comment by UnamSanctam** [comment]
> so i can leave my xmr wallet and only have to change port and password to mine ghostrider? is it also possible to mine on port 443?

Yes exactly, only thing you have to change to mine GhostRider on MoneroOcean is to add `~ghostrider` at the end of the password. And yes you can use port 443 just fine.

**Comment by harry133712** [comment]
okay thank you for ur quick response!
